### ISSUE TITLE
Write to wp-config.php

### ISSUE THREAD

**Issue by drywall** [webrain/grunt-wordpress-deploy#22]
Just thinking out loud: it might be nice if, instead of ignoring wp-config.php, this tool actually wrote to wp-config.php, updating it with the provided environment's database credentials and URLs (DB_NAME, DB_USER, DB_PASSWORD, DB_HOST, WP_HOME) during a file deploy. Since all that information is already present in-the-clear in the Gruntfile.js, why not provide a way to provide that information to the config file during a deployment?

**Comment by Grawl** [comment]
Completely agree. Good idea.

**Comment by herbie4** [comment]
I don't think this works for everybody. For me I use different wp-config settings, like switching on or off core updates, on local, staging and final.
